game.js

```diff
@@ -335,9 +335,11 @@ class Game {
   reset() {
     this.view = new View(this.squareSize);
     this.board = new Board(this.width, this.height);
-    this.isRunning = true;
-
+    this.isRunning = false;
+    this.generation = 0;
     this._addArmiesToBoard();
+    this.render();
```

Comment:
don't call the `render` in `reset`. `reset` means reset the game state and nothing else